internal/app/handlers.go

```diff
@@ -425,9 +425,9 @@ func (app *Application) handleBannerToggle(w http.ResponseWriter, r *http.Reques
 	action := r.PathValue("action")
 	var component templ.Component
 	if action == "close" {
-		component = components.UsaBanner()
+		component = components.UsaBanner("close")
```

Comment:
This looks great! You could simplify it even further by removing this if statement and just doing `component = components.UsaBanner(action)`!

Reply:
💡 done!